.travis.yml

```diff
@@ -15,6 +26,10 @@ before_install:
 script:
 - ./mvnw clean install
 
+after_success:
+- 'if [ "$TRAVIS_BRANCH" = "master" ] && [ "$TRAVIS_PULL_REQUEST" = "false" ]; then openssl aes-256-cbc -K $encrypted_9047eb4b7969_key -iv $encrypted_9047eb4b7969_iv -in cd/codesigning.asc.enc -out cd/codesigning.asc -d; gpg --fast-import cd/codesigning.asc; fi'
+- 'if [ "$TRAVIS_BRANCH" = "master" ] && [ "$TRAVIS_PULL_REQUEST" = "false" ]; then mvn deploy -Prelease --settings cd/settings.xml; fi'
```

Comment:
shouldn't we add skipTests so they won't run again?

Reply:
fixed